packages/e2e-tests/src/specs/dashboard/settings/adminUser/customFonts.js

```diff
@@ -79,8 +78,6 @@ describe('Custom Fonts', () => {
     await page.keyboard.press('ArrowDown');
     await page.keyboard.press('ArrowDown');
     await page.keyboard.press('ArrowUp');
-    const selected = await getSelectedFont();
```

Comment:
Later in this test we tab to the delete button which will end up being the same as looking for the selected font here.

```
 await expect(page).toMatchElement(
      `button[aria-label="Delete ${OPEN_SANS_CONDENSED_LIGHT}"]`
    );
```

We also have a specific test looking for font selection.